### DISCUSSION TITLE
error enrollment, help me!

### DISCUSSION THREAD

**Discussion by Exploiter-org** [3kh0/ext-remover#1359] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Does anyone know how to fix error enrollment

**Comment by schoolexploitkid** [comment]
powerwash again

**Comment by Exploiter-org** [comment]
It still says the same thing

**Reply by schoolexploitkid** [reply]
try using shimmer if it works

**Reply by Exploiter-org** [reply]
I don't want to unenroll

**Reply by ghost** [reply]
You don't have to un-enroll to use shimmer he wants you to use the shimmer re-enroll feature.

**Reply by Exploiter-org** [reply]
For what

**Reply by ghost** [reply]
To fix the enrollment error.

**Reply by Exploiter-org** [reply]
When I boot into sh1mmer what do I press

**Reply by schoolexploitkid** [reply, marked as answer]
re-enrollment (duh)
